interface/forms/dictation/view.php

```diff
@@ -12,6 +12,8 @@
 
 use OpenEMR\Core\Header;
 include_once("../../globals.php");
+require_once("../../../public/assets/htmlpurifier-4.9.2/library/HTMLPurifier.auto.php");
```

Comment:
Now I am not sure which line you are referring. If you are referring to this line:
require_once("/public/assets/htmlpurifier-4.9.2/library/HTMLPurifier.auto.php");
I am pretty sure it works without it because without that line the page loads. Please correct me if I am wrong but if the page makes a call to a class that it can't find that page should throw an error and not load, right?  Everything is working fine without that line. 
You can try it out at http://omp.openmedpractice.com/rolebase log in admin pass.